package.json

```diff
@@ -51,7 +51,7 @@
     "moonbase:export": "hardhat --network moonbase export --export exports/moonbase.json",
     "arbitrum:deploy": "hardhat --network arbitrum deploy",
     "arbitrum:export": "hardhat --network arbitrum export --export exports/arbitrum.json",
-    "test": "hardhat test test/_/*.ts test/abstract/*.test.ts test/constant-product/*.test.ts test/deployer/*.test.ts test/library/*.test.ts test/router/*.test.ts test/hybrid/*.test.ts",
+    "test": "hardhat test test/abstract/*.test.ts test/deployer/*.test.ts test/library/*.test.ts test/hybrid/*.test.ts",
```

Comment:
Reason to do this? I'm not worried about running extra tests as long as they pass

Reply:
this is cuz since we're changing the `TridentRouter` implementation a little bit (by disabling the call to bentobox), it will make some tests fail, like the constant-product and router tests, since they use the router directly